smtp.go

```diff
@@ -14,6 +14,9 @@ type Dialer struct {
 	Host string
 	// Port represents the port of the SMTP server.
 	Port int
+	// Identity is the host name sent to the SMTP server with the HELO command.
+	// The default is to use "localhost".
+	Identity string
```

Comment:
Could you rename that LocalName?
That is how the variable is called in [net/smtp] (https://golang.org/pkg/net/smtp/#Client.Hello). Also Identity can be confused with the identity variable of [smtp.PlainAuth](https://golang.org/pkg/net/smtp/#PlainAuth).